aggoracle/chaingerreader/evm.go

```diff
@@ -11,6 +11,16 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
+// InjectedGER is a struct that represents the injected GlobalExitRoot event
+type InjectedGER struct {
+	// BlockNumber is the block number of the event
+	BlockNumber uint64
+	// BlockIndex is the index of the event in the block
+	BlockIndex uint
```

Comment:
hmm, sorry for not commenting it in the other PR, but should this be `EventIndex`?

Reply:
I mean it could. On `aggkit` we call these fields block index or even block position. That is why I went with this.

Reply:
Yes it is strange (I don't like the way we are naming them in syncers, but it is what it is). It would be straightforward to call it as @vcastellm suggested.

Reply:
Since this PR already got merged on the prover side, lets leave it as is for now. I'll write this down so we can change it when we get the happy path working.